### PR TITLE
CalDAV Create events and search for events between dates

### PR DESCRIPTION
**Pull request by mariushegele**

### CODE REVIEW

File: services/cal/cal_service.py [hunk 1]
```diff
@@ -33,22 +40,29 @@ def __init__(self):
         if not all([var in environ for var in required_env]):
             raise EnvironmentError("Did not set all of these environmet variables: ", required_env)
```
Comment (Ovakefali13) [inline services/cal/cal_service.py]: The met is haunting you again

File: services/cal/event.py [hunk 2]
```diff
@@ -2,24 +2,23 @@
 from datetime import timedelta, datetime as dt
 from icalendar import Event as iCalEvent
 from icalendar import Alarm, vDatetime
+import pytz
 
 class Event(iCalEvent):
+    local_tz = pytz.timezone('Europe/Berlin')
```
Comment (Ovakefali13) [inline services/cal/event.py]: Timezone hardcoded ?

Comment (mariushegele) [reply services/cal/event.py]: Managed to do the same without. `.astimezone()` can be used to get system timezone.

**Comment by Ovakefali13** [comment]
Tests are still failing

**Pull request merged**